src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.impl.FruitStorageDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.FruitTransaction.Operation;
+import core.basesyntax.service.FileService;
+import core.basesyntax.service.FruitStorageService;
+import core.basesyntax.service.ParserService;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.impl.FileServiceCsvImpl;
+import core.basesyntax.service.impl.FruitStorageServiceImpl;
+import core.basesyntax.service.impl.ParserServiceImpl;
+import core.basesyntax.service.impl.ReportServiceCsvImpl;
+import core.basesyntax.strategy.BalanceOperationHandler;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.PurchaseOperationHandler;
+import core.basesyntax.strategy.ReturnOperationHandler;
+import core.basesyntax.strategy.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String FILE_NAME_SOURCE = "src/main/resources/file.csv";
```

Comment:
Avoid hardcoding file paths in your solution. It's better to pass file paths as command-line arguments or read them from a configuration file or environment variable. This will increase the flexibility and reusability of your code.